site/templates/items/title.jet

```diff
@@ -0,0 +1,10 @@
+{{block itemTitle(title,class)}}
+
+ {{titleImage := .ImageMap["Title"]}}
+
+  {{if isset(titleImage)}}
+    <img src="{{titleImage}}" alt="{{title}}" class="item-title-image {{class}}"></s72-image>
+  {{else}}
+    <h3>{{title}}</h3>
```

Comment:
this was a h3 for carousel titles and h1 for film detail title. im not sure whats semantically correct so ive just left this as h3 for now.